apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/nodes/TextResponseBlockNode/TextResponseBlock.spec.tsx

```diff
@@ -0,0 +1,91 @@
+import { MockedProvider } from '@apollo/client/testing'
+import { render, screen } from '@testing-library/react'
+import { Position, ReactFlowProvider } from 'reactflow'
+
+import { TreeBlock } from '@core/journeys/ui/block'
+
+import {
+  BlockFields_StepBlock as StepBlock,
+  BlockFields_TextResponseBlock as TextResponseBlock
+} from '../../../../../../../__generated__/BlockFields'
+import {
+  ThemeMode,
+  ThemeName
+} from '../../../../../../../__generated__/globalTypes'
+import { mockReactFlow } from '../../../../../../../test/mockReactFlow'
+import '../../../../../../../test/i18n'
+
+import { TextResponseBlockNode } from './TextResponseBlockNode'
+
+describe('TextResponseBlockNode', () => {
```

Comment:
The title of the file also needs to be changed to this `TextResponseBlockNode.spec.tsx`
```suggestion
describe('TextResponseBlockNode', () => {
```

Reply:
Oops, good catch!